addons/okr-daily-script.kp/knowledge.md

```diff
@@ -176,33 +176,46 @@ ms_install_days = ms\
 # combine data
 current = count_by_client_day\
   .join(ms_install_days, on='client_id', how='left')\
-  .join(ms_has_theme, on='client_id', how='left')\
+  .join(ms_has_theme, on=['client_id', 'normalized_channel'], how='left')\
   .drop('submission_date_s3')
 
 
 # add total number of distinct clients per day
 # and total number of ditscint clients with a custom theme
-n_clients = ms.select('client_id').distinct().count()
+n_clients = ms.select(['client_id', 'normalized_channel']).distinct()\
+               .groupby('normalized_channel').count()\
```

Comment:
It's possible that a given `client_id` could appear on multiple channels in the same day, which would mean counting the same client against more than one channel. 

Is that OK for downstream analysis? Or would it be better to pick a single `(client_id, normalized_channel)` for each client for the day first? Then the per-channel counts could be summed to compute accurate overall client counts.